image_test/metadata-script/shutdown-script-integrity.wf.json

```diff
@@ -3,14 +3,16 @@
   "Vars": {
     "source_image": {"Required": true, "Description": "Image to be tested"},
     "shutdown_msg": {"Required": true, "Description": "Hash of shutdown script"},
+    "wait_msg": {"Required": true, "Description": "Message to stop instance"},
     "shutdown_script_name": {"Required": true, "Description": "Shutdown script of the created instance"},
     "instance_url": "integrity-url",
     "instance_gcs": "integrity-gcs",
     "instance_public_gcs": "integrity-public-gcs",
     "instance_metadata": "integrity-metadata"
   },
   "Sources": {
-    "shutdown_file.ps1": "${shutdown_script_name}"
+    "shutdown_file.ps1": "${shutdown_script_name}",
+    "startup-script": "startup-script-for-shutdown-tests.sh"
```

Comment:
This will fail on windows, see above how the script is a variable, that way you pass it in for both windows and linux. The naming is a bit odd but just make this startup_script.ps1 (both windows and linux). Underscores are just what we use elsewhere, the extension just needs to be whatever scipt you use on windows (bat or ps1), linux will just ignore it.